apps/tasks/management/commands/rebuild_index.py

```diff
@@ -0,0 +1,38 @@
+from django.core.management import BaseCommand
+from apps.tasks.elasticsearch import elastic
+from rest_framework.renderers import JSONRenderer
+from apps.tasks.models import Task
+from apps.tasks.serializers import TaskESSerializer
+from elasticsearch import Elasticsearch
+
+
+class Command(BaseCommand):
+    help = 'Synchronises ElasticSearch indexes.'
+
+    def handle(self, *args, **kwargs):
```

Comment:
Nu e posibil de facut un serializer care sa iti returneze datele in formatul care trebuie iti trebuie?